# Ardumower Perimeter Message
# Value of single perimeter sensor

bool inside
int32 magnitude
bool signal_timeout



